### PR TITLE
Multiple gt pred pairs + CM update

### PR DESCRIPTION
**Pull request by Purg**
Change activity eval CLI to be able to take in multiple gt/pred file pairs and aggregate metrics/confusion-matrix/PR/ROC outputs over all inputs. Maintains per-pair output of pred-over-GT bar plot output.

Also updated confusion matrix output as discussed. Adding use of seaborn.

### CODE REVIEW

**Review by hdefazio — APPROVED** [review]

File: angel_system/ptg_eval/activity_classification/evaluate_activity.py [hunk 1]
```diff
@@ -13,28 +17,68 @@
...
-    # implementation.
-    gt = activities_as_dataframe(gt)
-    detections = activities_as_dataframe(detections)
+    time_window = args.time_window
+    uncertainty_pad = args.uncertainty_pad
+
+    # TODO: For multiple gt/pred pairs, loop over the following 3 sections,
+    #       vertically stacking the `gt_true_mask` and `dets_per_valid_time_w`
+    #       matrices.
+    # TODO: Assert that the labels returned for each pair is equal.
```
[the remark this answers is not in the document]
Comment (hdefazio) [reply angel_system/ptg_eval/activity_classification/evaluate_activity.py:34]: > TODO: Assert that the labels returned for each pair is equal.

Is this talking about consistency between gt and detections? Or what is  already here (consistency between pairs)

[the remark this answers is not in the document]
Comment (Purg) [reply angel_system/ptg_eval/activity_classification/evaluate_activity.py:34]: I forgot to remove these lines. Will remove.

[the remark this answers is not in the document]
Comment (hdefazio) [reply angel_system/ptg_eval/activity_classification/evaluate_activity.py:34]: Ok, consider it approved once that is pushed

[the remark this answers is not in the document]
Comment (Purg) [reply angel_system/ptg_eval/activity_classification/evaluate_activity.py]: Fixed and rebased. Merging.

**Pull request merged**